extra-utils/peazip/autobuild/build

```diff
@@ -0,0 +1,52 @@
+if   [[ "${CROSS:-$ARCH}" = "amd64" ]]; then
+    ARCH_OPTS="x86_64"
+elif [[ "${CROSS:-$ARCH}" = "arm64" ]]; then
+    ARCH_OPTS="aarch64"
+fi
+
+abinfo "Building PeaZip by Lazarus..."
+lazbuild \
+  --verbose \
+  --lazarusdir=/usr/share/lazarus \
+  --cpu=${ARCH_OPTS} \
+  --widgetset=gtk2 \
+  --max-process-count=$(nproc --all) \
+  -B project_pea.lpi project_peach.lpi
+
+abinfo "Installing PeaZip..."
+mkdir -v "${PKGDIR}/usr/bin" -p
+mkdir -v "${PKGDIR}/usr/lib/peazip/res" -p
+mkdir -v "${PKGDIR}/usr/lib/peazip/res/lang" -p
+mkdir -v "${PKGDIR}/usr/share/pixmaps" -p
+mkdir -v "${PKGDIR}/usr/share/doc/peazip" -p
+mkdir -v "${PKGDIR}/usr/share/applications" -p
+mkdir -v "${PKGDIR}/usr/share/kservices5/ServiceMenus" -p
+
+install -v -m755 "${SRCDIR}/peazip" "${PKGDIR}/usr/lib/peazip/peazip"
+install -v -m755 "${SRCDIR}/pea" "${PKGDIR}/usr/lib/peazip/pea"
+install -v -m644 "${SRCDIR}/../peazip_help-${PKGVER}.pdf" "${PKGDIR}/usr/share/doc/peazip/peazip_help.pdf"
+install -v -m644 "${SRCDIR}/FreeDesktop_integration/peazip.png" "${PKGDIR}/usr/share/pixmaps/peazip.png"
+install -v -m644 "${SRCDIR}/FreeDesktop_integration/peazip.desktop" "${PKGDIR}/usr/share/applications/peazip.desktop"
+
+pushd "${PKGDIR}/usr/bin"
+ln -sv /usr/lib/peazip/peazip ./peazip
+ln -sv /usr/lib/peazip/pea ./pea
+popd
+
+pushd "${PKGDIR}/usr/lib/peazip"
+for i in 7z brotli zpaq zstd upx unrar unace
+do
+  mkdir -v "${PKGDIR}/usr/lib/peazip/res/$i"
+  ln -sv /usr/bin/$i ./res/$i/$i
```

Comment:
Same as above.

Reply:
It's use system's exec binary. I think it should be absolute path.

Reply:
This is irrelevant. If you set this as absolute path, then when running the binary in a non-current sysroot would result in possible file-not-found situations.